### DISCUSSION TITLE
Simbrief FLightplans Resulting In Sluggish Climb Performance & Running Out Of Fuel

### DISCUSSION THREAD

**Discussion by muskysim** [flybywiresim/aircraft#5382] [Help] [answered]
The last couple of updates to the Development version seemed to have broken the A320NX for me specifically when using Simbrief flight plans.

Yesterday using Simbrief plan loaded I noticed very sluggish climb performance in managed mode, it was barely 400/500fpm. Any increase in selected VS mode and speed would just bleed away. Also halfway through the flight it ran out of fuel and this has never happened before with a loaded Simbrief flight plan.

Tried flight again and got the same problem with sluggish climb performance and again ran out of fuel.

Updated Development version again today but encountered a more severe issue. When loading Simbrief flight plan in the INIT page of the FMGS, MSFS would hang and not respond to any further mouse click.

Tried this 3 times and every time after loading Simbrief flight plan MSFS would hang. Also tried deleting folder and performing clean install but got the same result.

Tried loading flight plan from MSFS and encountered none of the issues described above. Confirming to me that it seems to be Simbrief flight plan related.

One other difference I noticed was I was not required to double-click ZFW to load it in FMGS or manually enter Fuel which I used to have to do.

**Comment by p493z3r0** [comment]
Regarding clb performance: Are you sure you retracted flaps and landing gear?

**Reply by muskysim** [reply]
Yes I have been caught out by that mistake in the  past so I triple checked that it was clean config but behaviour is exactly like the landing gear is extended.

**Reply by NUTTTTTTTT** [reply]
MSFS live weather is currently bugged causing issues with aircraft performance, airspeed/climb rate etc- If possible use a static weather preset for now. You can usually identify it from temps on your lower ECAM, but most people are stuck with it at the moment.

**Reply by HankL340i** [reply]
I have experienced something similar in the last few development builds. I currently have build 654061f installed. The MCDU is showing optimal FL in the 230 to 250 range early in the flight but increases to 370-ish a bit later in the flight. More important, the aircraft won't climb above say FL270. I can force the climb using EXPIDITE; however, once I reach the desired FL, the aircraft is very slow to accelerate. TOW has pretty much always been in 68T range. I have not looked at fuel use. I am using a SIMBRIEF flight plan and have also noticed that the process of loading the plan into the MCDU varies. Sometimes the MCDU entries are blank and I have the INIT REQ key while other times the plan is mostly present in the MCDU and there is no INIT REQ option. Also, ZFW in the MCDU does not match the SIMBRIEF OFP.

**Reply by NUTTTTTTTT** [reply]
@HankL340i Please read my comment above. Check your lower ECAM temps to verify and try a preset. The default simbrief OFP is not set up to match the FBW, you can use the custom profile for that.

**Reply by HankL340i** [reply]
Will do this, Thanks!

**Comment by NUTTTTTTTT** [comment, marked as answer]
Answering to main thread rather than other answers - MSFS live weather is currently bugged causing issues with aircraft performance, airspeed/climb rate etc- If possible use a static weather preset for now. You can usually identify it from temps on your lower ECAM, but most people are stuck with it at the moment.

**Reply by muskysim** [reply]
Thanks for that I certainly have enough now to work with.to investigate further and will report what I find but live weather makes sense. Noticed some weird turbulence effects

**Comment by 2hwk** [comment]
There is currently (as of the time of this post) an issue with live weather that has persisted over the last 24 hours where the outside temperatures will be too high vs what it should be in reality which will affect engine performance, resulting in reduced performance and increased fuel consumption.

Use a weather preset until this is fixed by Asobo

**Reply by HankL340i** [reply]
NUTTTTTTTT and 2hwk, Thanks, I had no issue using a weather preset. One less shortcut I'll be taking (not checking the temperature)!

**Comment by AdamNZ** [comment]
As a matter of interest, I use Unreal Weather/Live Metar and also seem to get this problem albeit only occasionally. I'm wondering whether UWLM "inherits" the Asobo bug <?>.
